src/runtime/virtcontainers/sandbox.go

```diff
@@ -2217,6 +2217,46 @@ func (s *Sandbox) GetAgentURL() (string, error) {
 	return s.agent.getAgentURL()
 }
 
+// GuestVolumeStats return the filesystem stat of a given volume in the guest.
+func (s *Sandbox) GuestVolumeStats(ctx context.Context, volumePath string) ([]byte, error) {
+	guestMountPath, err := s.guestMountPath(volumePath)
+	if err != nil {
+		return nil, err
+	}
+	return s.agent.getGuestVolumeStats(ctx, guestMountPath)
+}
+
+// ResizeGuestVolume resizes a volume in the guest.
+func (s *Sandbox) ResizeGuestVolume(ctx context.Context, volumePath string, size uint64) error {
+	// TODO: invoke the resize block device feature provided by the hypervisor.
+	// if err := s.hypervisor.ResizeBlockDevice(ctx, devicePath, size); err != nil {
+	// 	   return err
+	//}
```

Comment:
Please remove commented out code and add a TODO comment with a link to a GitHub issue if required.